addons/webhook/certificates.go

```diff
@@ -0,0 +1,125 @@
+package webhook
+
+import (
+	"context"
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	certutil "k8s.io/client-go/util/cert"
+	"k8s.io/client-go/util/keyutil"
+	rsrcs "knative.dev/pkg/webhook/certificates/resources"
+	"path"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"time"
+)
+
+const (
+	ServerKey = "server-key.pem"
+	// ServerCert is the name of the key associated with the secret's public key.
```

Comment:
This description here seems to belong to ```ServerKey```. Doesn't it?

Reply:
correct. Removing comment. is redundant.